src/docfx/build/xref/IXrefSpec.cs

```diff
@@ -14,5 +14,7 @@ internal interface IXrefSpec
         HashSet<string> Monikers { get; }
 
         string GetXrefPropertyValueAsString(string propertyName);
+
+        ExternalXrefSpec ToExternalXrefSpec();
```

Comment:
suggest to remove

Reply:
with this, we do not need to check if `InternalXrefSpec`